### PR TITLE
added lowercase functionality for all commands and fixed params

### PR DESCRIPTION
**Pull request by williamlamjy**

### CODE REVIEW

File: src/main/java/seedu/mtracker/console/AddInstrumentParser.java [hunk 1]
```diff
@@ -64,9 +64,9 @@ public static String getInstrumentSentimentFromUser() {
 
 
     public static void addSentimentToParameters() {
-        String sentiment = getInstrumentSentimentFromUser();
+        String sentiment = getInstrumentSentimentFromUser().toLowerCase();
         while (!Validate.isValidSentiment(sentiment)) {
-            sentiment = getInstrumentSentimentFromUser();
+            sentiment = getInstrumentSentimentFromUser().toLowerCase();
         }
```
Comment (KVignesh122) [inline src/main/java/seedu/mtracker/console/AddInstrumentParser.java:70]: Change to do-while loop.

File: src/main/java/seedu/mtracker/console/InputParser.java [hunk 2]
```diff
@@ -138,7 +138,7 @@ public FindCommand getFindInstrumentsCommand(String[] commandComponents)
     public Command filterByCommandType(String[] commandComponents, ArrayList<Instrument> instruments)
             throws Exception {
         Command command;
-        switch (commandComponents[MAIN_COMMAND_INDEX]) {
+        switch (commandComponents[MAIN_COMMAND_INDEX].toLowerCase()) {
```
Comment (KVignesh122) [inline src/main/java/seedu/mtracker/console/InputParser.java:141]: Good thinking!

**Review by KVignesh122 — COMMENTED** [review]
Mostly good. Change one of your edits to do-while loop, then would be good for merging. Remember to update branch.

File: src/main/java/seedu/mtracker/console/AddInstrumentParser.java (hunk 1 above)
Comment (williamlamjy) [reply src/main/java/seedu/mtracker/console/AddInstrumentParser.java:70]: i think can merge theodore's pr on do while loop then ill merge this after or the other way arnd

Comment (KVignesh122) [reply src/main/java/seedu/mtracker/console/AddInstrumentParser.java:70]: Ok yours can go first then.

**Pull request merged**